jquery.flot.js

```diff
@@ -630,26 +644,34 @@ Licensed under the MIT license.
                     // find out how to copy
                     format.push({
```

Comment:
using objects with the same signature for format significantly improves performance as it prevents deoptimization